Production/govuk_ios/ViewControllers/SearchViewController.swift

```diff
@@ -230,6 +236,12 @@ class SearchViewController: BaseViewController,
             snapshot.appendItems(results, toSection: .results)
         }
         dataSource.apply(snapshot, animatingDifferences: true)
+
+        if viewModel.results?.isEmpty == true || viewModel.results == nil {
```

Comment:
We could one line this also?

tableView.isHidden = viewModel.results?.isEmpty == true || viewModel.results == nil